back/contact/ContactController.js

```diff
@@ -3,36 +3,47 @@ let router = express.Router();
 let bodyParser = require('body-parser');
 const nodemailer = require('nodemailer');
 
-router.use(function (req, res, next) {
+/*router.use(function (req, res, next) {
     res.header("Access-Control-Allow-Origin", "*");
     res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
     res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
     next();
-});
+});*/
 router.use(bodyParser.urlencoded({ extended: true }));
-router.use(bodyParser.json());
+//router.use(bodyParser.json());
 
 
 const transporter = nodemailer.createTransport({
     service: 'gmail',
     auth: {
         user: 'hgloves.ftc@gmail.com',
-        pass: 'hytVND25'
+        pass: 'gYFR84fe9'
     }
 });
 
+const mailOptions = {
+	from: 'hgloves.ftc@gmail.com',
+	to: 'nicolas.albiges@epitech.eu',
```

Comment:
Not sure here but at least one of this mail shouldn't be hard coded ?

Reply:
Nope because we want to know send and receive the mail they want to send us. Their email adress is in the title of the mail, ask me if you want more information.

Reply:
And here i put my adress mail for testing purposes.